pallets/did/src/tests.rs

```diff
@@ -23,23 +23,23 @@ use sp_std::{collections::btree_set::BTreeSet, convert::TryFrom};
 use crate::{self as did, mock::*};
 use ctype::mock as ctype_mock;
 
-// submit_did_create_operation
+// create
```

Comment:
i guess this comment doesn't add much value.

Reply:
It separates the different extrinsics that are tested here. Do you think it's not useful?

Reply:
It creates sections? In objective-c you can [make sections in code](https://stackoverflow.com/questions/6853821/what-is-the-significance-of-pragma-marks-why-do-we-need-pragma-marks) which is really cool :D Didn't recognized it as a section. Thought it was only for the next test